snare/tests/test_add_meta_tag.py

```diff
@@ -28,5 +28,8 @@ def test_add_meta_tag(self):
         assert(soup.find("meta", attrs={"name": "google-site-verification"}) and
                soup.find("meta", attrs={"name": "msvalidate.01"}))
 
+        config['WEB-TOOLS'] = dict(google='', bing='')
```

Comment:
Usually, different input conditions for a method are tested using separate test methods, so you can have a method name like `test_add_meta_tag_with_empty_tags` or anything else.